pkgs/development/tools/misc/universal-ctags/default.nix

```diff
@@ -1,17 +1,17 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, perl, pythonPackages, libiconv, jansson }:
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, perl, python3Packages, libiconv, jansson }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "universal-ctags";
-  version = "unstable-2019-07-30";
+  version = "p5.9.20210124.0";
```

Comment:
`p` here is a bit confusing, since I cannot do uninstall of `universal-ctags`. Not sure what's the right way to address this.

Reply:
The version number is probably just `5.9.20210124.0` and the p can be added when fetching.

Reply:
That was my guess as well, updated.